packages/embedded_ocaml_templates/embedded_ocaml_templates.0.8/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis:
+  "EML is a simple templating language that lets you generate text with plain OCaml"
+description:
+  "EML is a simple templating language that lets you generate text with plain OCaml"
+maintainer: ["Emile Trotignon <emile.trotignon@gmail.com>"]
+authors: ["Emile Trotignon <emile.trotignon@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/EmileTrotignon/embedded_ocaml_templates"
+bug-reports:
+  "https://github.com/EmileTrotignon/embedded_ocaml_templates/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "3.6" & >= "2.8.0"}
```

Comment:
```suggestion
  "dune" {>= "3.6"}
```
You can avoid this duplication by removing dune from the dependencies of the project in the dune-project file